rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/repositories/BaseRepositoryTest.scala

```diff
@@ -30,7 +30,7 @@ import scala.concurrent.{Await, Future}
 
 abstract class BaseRepositoryTest extends AnyWordSpec with TestContextManagement with BeforeAndAfter {
 
-  val awaitDuration: Duration = Duration(2000, TimeUnit.MILLISECONDS)
+  val awaitDuration: Duration = Duration(8000, TimeUnit.MILLISECONDS)
```

Comment:
(why/) was it needed?

Reply:
Probably I have included that one by mistake. However, with 2000 await duration integration tests are always failing on my machine, and with 8000 they work fine (on any branch). So maybe this change can stay there.

Good catch tho.

Reply:
if that's the case then I suggest let's keep it